src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const hasAge = age && <p className="Person__age">{`I am ${age}`}</p>;
```

Comment:
2 points:
- no need to assign it to a variable
- if you need this variable, change name, `has...` prefix makes it look like `hasAge` is a boolean, however, it's not

Reply:
fixed